sandbox-backend/src/main/java/com/exadel/sandbox/team5/dao/util/SearchCriteria.java

```diff
@@ -0,0 +1,30 @@
+package com.exadel.sandbox.team5.dao.util;
+
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+
+import java.awt.print.Pageable;
+
+public class SearchCriteria {
+    private int page;
```

Comment:
I prefare "pageNum", "pageSize" naming. @VladimirZhuk07 thoughts?

Reply:
We used on one project the following:
```
    private int itemsPerPage;
    private int pageNumber;
...

Reply:
Also let's move this parameters to separate nested object using for example custom `Paging` class. Why I think that we should create separate object it will be easier to reuse it in the future if it will be required, also we cannot have pagable object `pageNumber` without `itemsPerPage`. I would like to have such logic in the separate class using for example special constructors.

Reply:
Okay.